.pre-commit-config.yaml

```diff
@@ -0,0 +1,36 @@
+repos:
+-   repo: https://github.com/python-poetry/poetry
+    rev: 1.6.1
```

Comment:
Do we have to set rev’s here or can we rely on the ones set in the pyproject.toml file?

Reply:
I haven't found a way to do this yet